### PR TITLE
Collapsible brightness table

### PR DESCRIPTION
**Pull request by rpiaggio**
Kapture 2022-12-14 at 14 38 26

### CODE REVIEW

File: common/src/main/webapp/sass/explore.scss [hunk 1]
```diff
@@ -25,6 +25,11 @@ $tree-section-width: 300px;
   background: var(--control-background-color);
 }
 
+// Should we just set this in the "lucuma" styles?
+.p-accordion .p-accordion-header:not(.p-disabled) .p-accordion-header-link:focus {
+  box-shadow: none !important;
```
Comment (toddburnside) [inline common/src/main/webapp/sass/explore.scss:30]: We should probably put it in the lucuma styles. Can it be done without the `!important`?

Comment (rpiaggio) [reply common/src/main/webapp/sass/explore.scss:30]: I expect the !important not to be needed if it's in the base style. I'll remove from here.

File: explore/src/main/scala/explore/targeteditor/SpectralDefinitionEditor.scala [hunk 2]
```diff
@@ -85,16 +88,19 @@ sealed abstract class SpectralDefinitionEditorBuilder[
...
-  protected val disabledItems: HashSet[SEDType[T]]
+  protected def currentType: SpectralDefinition[T] => SEDType[T]
+  protected def disabledItems: HashSet[SEDType[T]]
 
   val component = ScalaFnComponent
     .withHooks[Props]
     .useContext(AppContext.ctx)
-    .render { (props, ctx) =>
+    // .useStateView(IsExpanded(true))
+    .render { (props, ctx) => // , brightnessExpanded) =>
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/SpectralDefinitionEditor.scala:103]: Commented code

**Review by toddburnside — APPROVED** [review]
LGTM

**Comment by cquiroz** [comment]
Looks very nice

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**